src/user/user.service.ts

```
@@ -1,70 +1,70 @@
-        } catch (error) {
-          if (error.code === 'ER_DUP_ENTRY') {
-            throw new ConflictException('Email already exists')
-          };
-        }
+  constructor(
+    @InjectRepository(User)
+    private usersRepository: Repository<User>,
+  ) { }
+
+  // Create user
+  async createUser(inputUser: CreateUserDto): Promise<Object> {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)